### PR TITLE
Cigar Guides imagecta

### PR DESCRIPTION
**Pull request by kailasnadh790**
part of https://github.com/hlxsites/famous-smoke-cigaradvisor/issues/131
Test URLs:
- Before: https://main--famous-smoke-cigaradvisor--hlxsites.hlx.page/cigaradvisor/cigar-guides
- After: https://bugfix-50-50-guides--famous-smoke-cigaradvisor--hlxsites.hlx.page/cigaradvisor/cigar-guides

### CODE REVIEW

File: cigaradvisor/styles/styles.css [hunk 1]
```diff
@@ -248,7 +248,6 @@ main .section[data-layout] .layout-wrapper > div {
   display: flex;
   flex-direction: column;
   align-items: stretch;
-  justify-content: space-between;
```
Comment (bstopp) [inline cigaradvisor/styles/styles.css:251]: This has more implications than just fixing the CTA wrapper

**Review by bstopp — CHANGES_REQUESTED** [review]

File: cigaradvisor/styles/styles.css (hunk 1 above)
Comment (kailasnadh790) [reply cigaradvisor/styles/styles.css]: modified. pls review.

File: cigaradvisor/blocks/imagecta/imagecta.css [hunk 2]
```diff
@@ -23,9 +23,16 @@
 
 .section[data-layout="50/50"] .imagecta-wrapper{
   display: flex;
+}
+
+.section[data-layout="50/50"].article-teaser-container .imagecta-wrapper{
   flex-basis: 100%;
 }
 
+.section.imagecta-container.separator-container:not([class*=""]) .layout-wrapper > div {
```
Comment (bstopp) [inline cigaradvisor/blocks/imagecta/imagecta.css:32]: Can you get me an example of what this is trying to fix? Maybe i'm not understanding the need?

Comment (kailasnadh790) [reply cigaradvisor/blocks/imagecta/imagecta.css:32]: Sure, when the number of imageCTAs are not same on left and right grids of a 2-col layout, imageCTA are getting stretched to occuppy the available space..which is not how it looks on the live site.
image

**Review by bstopp — APPROVED** [review]

**Pull request merged**